requirements/bindep_requirements.txt

```diff
@@ -0,0 +1,2 @@
+subversion [platform:rpm]
+subversion [platform:dpkg]
```

Comment:
@rooftopcellist has reminded me that `git` needs to be added too

there might also be a better way to write this, like `subversion [platform:rpm platform:dpkg]`